### ISSUE TITLE
Translations Needed

### ISSUE THREAD

**Issue by corbinlc** [CypherpunkArmory/UserLAnd#430]
**Is your feature request related to a problem? Please describe.**
Many of our users are non English speakers, but like many apps we have only supported English.  As of today we now have translations for Chinese and Persian added to our repo, but there are so many more languages that would help.  For example, we have more installs in Brazil than any other country, but we don't have a Portuguese translation.  

**Describe the solution you'd like**
I would like anyone to get involved and start contributing translations for any language that they are fluent in.  Doing so is fairly easy as most of the text is found in [one file](https://github.com/CypherpunkArmory/UserLAnd/blob/staging/app/src/main/res/values/strings.xml) but we will help you as much as needed to make this happen.

**Describe alternatives you've considered**
We don't want to pay for this service because we want someone who is technical and part of our community to partner with us in bringing UserLAnd to your native tongue.

**Comment by AuntieE** [comment]
Hi,
   I am a native English speaker, but work as a member/admin of the French translation team of Full Circle Magazine (https://fullcirclemagazine.org ; www.fullcirclemag.fr) devoted to Linux Ubuntu in all its forms and, more particularly to Ubuntu.
  Have I understood correctly that what you need is a translation of the commands in your file? That would be possible I think. Just let me know if you think I can help. In a former life, I taught translation from French to English to graduate students at the Sorbonne. Translation theory is the same in the other direction...

**Comment by dragonSilverString** [comment]
I think you are perfectly qualified for translating the string.xml to french given your job, your skills and your experience using our app.  We would love to have you take a crack at it.

**Comment by AuntieE** [comment]
OK! Will do my best.

Provenance : Courrier<https://go.microsoft.com/fwlink/?LinkId=550986> pour Windows 10

________________________________
De : dragonSilverString <notifications@github.com>
Envoyé : Wednesday, November 28, 2018 6:20:16 PM
À : CypherpunkArmory/UserLAnd
Cc : AuntieE; Manual
Objet : Re: [CypherpunkArmory/UserLAnd] Translations Needed (#430)


I think you are perfectly qualified for translating the string.xml to french given your job, your skills and your experience using our app. We would love to have you take a crack at it.

**Comment by corbinlc** [comment]
I think you are perfectly qualified for translating the string.xml to french given your job, your skills and your experience using our app. We would love to have you take a crack at it.

**Comment by corbinlc** [comment]
Sorry, my daughters github account was logged in.  Made the same comment now using my name.

**Comment by EnderNightLord-ChromeBook** [comment]
is Chinese supported? I Can Help With That

**Comment by dragonSilverString** [comment]
We just merged this.  Would you remind reviewing it and suggesting any
modifications you think are warranted.
https://github.com/CypherpunkArmory/UserLAnd/blob/staging/app/src/main/res/values-zh-rCN/strings.xml

**Comment by tsuchiura** [comment]
I'd make Japanese version if you need it.

**Comment by corbinlc** [comment]
@tsuchiura please do.  Tell us if you need any assistance.

**Comment by EnderNightLord-ChromeBook** [comment]
Hey corbin is there any languages you want added?

**Comment by corbinlc** [comment]
I simply want people to know that helping out with translations would be very appreciated.  This can be any language they are fluent with.

**Comment by foodmaher** [comment]
@corbinlc
Hello I would like to translate the application into Arabic so I can translate it and I can send you a strings.xml file in Arabic and you are merging it with the program

**Comment by corbinlc** [comment]
Thanks.  That would be appreciated.

**Comment by foodmaher** [comment]
@corbinlc 

> Thanks. That would be appreciated.

File strings.xml translator for arabic ready how to upload it to you can you receive it by Mega or how

**Note:** It contains texts that are not translated into Arabic because they contain variable names that I can not type because Arabic is written from the right side to the left (unlike the English language). Here is the problem of invoking the names of the variables that will make them not work so I put them as they are until I modify them on the computer And upload it to you again and also it may contain the wrong translation that will be corrected in the future.

Thanks .. 👍👍

**Comment by foodmaher** [comment]
@corbinlc 
@MatthewTighe 
This is an Arabic language file but needs future modifications after its merge with the application to see if the sentences are consistent and understandable for users : [Arabic language.zip](https://github.com/CypherpunkArmory/UserLAnd/files/2831206/Arabic.language.zip)

**Comment by Anton2365** [comment]
I'm willing to do a German translation, if needed. :)

**Comment by corbinlc** [comment]
We would definitely appreciate that.  Tell us if you need any help.

**Comment by Atavic** [comment]
I copied [this](https://github.com/CypherpunkArmory/UserLAnd/blob/433635b95742b15523a142afa9f3c9417a960a1a/termux-app/terminal-term/src/main/res/values/strings.xml) to [this gist](https://gist.github.com/Atavic/4cbf89be7b5ec448c659d90ccd11e4a5) with italian translation.

Some notes: I used *script* for *transcript*, as script is mostly used by italians. 
I used *richiami di esecuzione* for *intents*, but it is a rarely used term, maybe the english term could be included, as:  

richiami di esecuzione (intents)

...like I did for: 

risveglio del terminale (wakelock)

**Comment by lewisdonofrio** [comment]
https://github.com/CypherpunkArmory/UserLAnd/issues/639

**Comment by AndrewScibek** [comment]
@s8321414 Feel free to look at what is already there. I assume it can be improved quite a bit. It was mostly just 1 for 1 translated from simplified Chinese.

**Comment by sujunmin** [comment]
Hi @s8321414 @AndrewScibek I found out the Tradition Chinese strings are still not translated manually, so I do the translation for Tradition Chinese, #686 .

**Comment by AndrewScibek** [comment]
@sujunmin That is correct they were just google translated from simplified chinese

**Comment by sujunmin** [comment]
@AndrewScibek Yes, the translation I created PR is included for native regular usage.

**Comment by s8321414** [comment]
In fact, I am going to rework it :3......

**Comment by thezakman** [comment]
I can make the Brazilian Portuguese translation with no problem. ;)

**Comment by corbinlc** [comment]
That would be great!  Thanks!

**Comment by EnderNightLord-ChromeBook** [comment]
that will be awesome... finally...

**Comment by thezakman** [comment]
I'm done with the translation.

**Comment by BestPinguin** [comment]
I'm a French native and I have a medium level in English, I can also help if you're interested...

**Comment by Jzarecta** [comment]
I recommend to use transifex to manage the translations

**Comment by EnderNightLord-ChromeBook** [comment]
Yo... a Spanish translation has already been added.

**Comment by qqqwwweee123-arch** [comment]
Long time no translation

**Comment by rodrigoesborges** [comment]
I can help with portuguese, spanish and, maybe french translation teams

**Comment by corbinlc** [comment]
Please feel free to submit a pull request with modifications to the existing translations.

**Comment by khaiphan9x** [comment]
Vietnamese

**Comment by luongthomas** [comment]
Hi @khaiphan9x, can you help with Vietnamese?

**Comment by seisdr** [comment]
Why the Arabic language is not full implement

**Comment by Ti64CLi** [comment]
I can help with the French translation if needed

**Comment by corbinlc** [comment]
@seisdr I think it is because the app changed after the last time someone worked on that translation.  Feel free to contribute the missing parts of the translation.

**Comment by corbinlc** [comment]
@Ti64CLi  feel free to take a crack at it.  There is an incomplete translation currently, because the app changed after the last Frech translation work.

**Comment by Matthellda** [comment]
Add Filipino Language please?

**Comment by krisfremen** [comment]
Hey there,

I'd be willing to contribute Macedonian translations!

Will start translating soon.

**Comment by onceagainifindmyselfhere** [comment]
I could be wrong but issue reporting on github in general could do with some translation. People aren't able to set proper reports because they cant read the instructions

**Comment by saed-rr** [comment]
#1638

**Comment by ElGlicho** [comment]
@saed-rr What's the matter? why did you mention my issue?

**Comment by Mohsenkalbarii** [comment]
Hello. My language is Persian. I am not that familiar with the software. I am very confused. Please forgive me.

**Comment by ye620** [comment]
Hack cmd

**Comment by Bajbus87aik** [comment]
Danke euch allen von ganzem Herzen ❤️

**Comment by iambaidar** [comment]
I am kashmiri.

**Comment by jhanshahyahmd645-lang** [comment]
چرا وقتی وارد برنامه userland میشم ازمن کد و پسورد username نمیخواد

**Comment by thetwindawei-alt** [comment]
Ok

**Comment by sifeddineakhouzine24-svg** [comment]
good afternoon please i need help i have hacker from my phone please choking my phone

**Comment by sifeddineakhouzine24-svg** [comment]
good afternoon please i need help i have hacker from my phone please choking my phone

**Comment by sifeddineakhouzine24-svg** [comment]
good afternoon please i need help i have hacker from my phone please choking my phone

**Comment by abdilany6688-sys** [comment]
كامل ادوات تثبيت كالي لينكس في الهاتف

**Comment by zubairahmedbaloch723-svg** [comment]
Dill

**Comment by zubairahmedbaloch723-svg** [comment]
Dill

**Comment by zubairahmedbaloch723-svg** [comment]
Zubair

**Comment by buenoribeiroerika17-art** [comment]
Como abrir uma tela

**Comment by corbinlc** [comment]
Closing as part of a repository cleanup. If this issue is still relevant, feel free to reopen it or file a fresh one.
